server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/DatafileManager.java

```diff
@@ -329,23 +329,34 @@ void bringMinorCompactionOnline(TabletFile tmpDatafile, TabletFile newDatafile,
     StoredTabletFile newFile;
     // rename before putting in metadata table, so files in metadata table should
     // always exist
+    boolean attemptedRename = false;
     do {
       try {
         if (dfv.getNumEntries() == 0) {
+
           tablet.getTabletServer().getFileSystem().deleteRecursively(tmpDatafile.getPath());
-        } else {
-          if (tablet.getTabletServer().getFileSystem().exists(newDatafile.getPath())) {
+        } else // i.e. dfv.getNumEntries() > 0
+        {
```

Comment:
@tushardhadiwal some minor comments:
* Please modify the placement of this curly brace to match the project's expected style, otherwise the CI fails. The [patch submission steps](https://github.com/apache/accumulo/blob/master/CONTRIBUTING.md#patch-submission) are very useful - ```mvn clean verify -DskipTests``` will usually auto-format code and fix this
* Related to above - personally I'd also remove the // i.e. dfv.getNumEntries() > 0 comment :-)

Reply:
@arvindshmicrosoft I have removed the comment & fixed the curly brace.